src/tools/pontoon/teams_projects.md

```diff
@@ -2,69 +2,68 @@
 
 <!-- toc -->
 
-The [Teams page](https://pontoon.mozilla.org/teams/) lists all locales that are enabled in Pontoon. From here you can access a specific [Team page](#team-page), where you’ll get a list of all projects enabled for the requested locale. Selecting a project from the *Team page* will take you to what is called the [Localization page](#localization-page).
-
-From each locale’s *Team page* it’s also possible to [request a project](#requesting-a-project), that is asking to add that locale to a project already available in Pontoon, not asking to support localization for a brand new project.
-
-The [Projects page](https://pontoon.mozilla.org/teams/) lists all projects that are available in Pontoon. From here you can access the list of locales that are enabled for the requested project. This is called a [Project page](#project-page) in Pontoon. By selecting a locale in the *Project page* you access the [Localization page](#localization-page).
-
-There is also a special Localization page that allows to access all resources for all projects enabled for a specific locale: `/locale/all-projects/all-resources`.
+Pontoon organizes localizations by locale/project pairs. Pontoon's structure reflects this with separate pages for [Teams](#teams) (locales), [Projects](#projects) and [Localizations](#localization-page).
 
 The following diagram shows how these pages are organized and connected to each other:
 
 ![Team and project pages flow](../../assets/images/pontoon/teams_projects/page_flow.png)
 
-## Team page
+## Teams
+
+### Teams page
 
-You can access a team’s page by selecting it from the [Teams page](https://pontoon.mozilla.org/teams/) or by appending `/YOUR_LOCALE_CODE` to the URL, e.g. `https://pontoon.mozilla.org/it` for Italian.
+The Teams page lists all locales that are enabled in Pontoon. It can be reached by clicking `Teams` from the navbar or by using the `/teams` URL (e.g. [pontoon.mozilla.org/teams](https://pontoon.mozilla.org/teams/)). Selecting a specific [Team page](#team-page) shows a list of all projects enabled for the requested locale. Choosing one of these projects leads to the [Localization page](#localization-page).
```

Comment:
```suggestion
The Teams page lists all locales that are enabled in Pontoon. It can be reached by clicking `Teams` from the navbar or by using the `/teams` URL (e.g. [pontoon.mozilla.org/teams](https://pontoon.mozilla.org/teams/)). Clicking on a specific locale will open the locale’s [Team page](#team-page).
```

I would move the "localization page" part into the team page section. Same for the Project page blurb.